src/main/java/Week8Homework/Athlete.java

```diff
@@ -0,0 +1,104 @@
+package Week8Homework;
+
+public class Athlete extends AthleteRankingComparator{
```

Comment:
```suggestion
public class Athlete {
```
the Athlete shouldn't extend the comparator. If you really want it, Athlete should actually implement `Comparable<Athlete>`, but a data class extending a comparator is not recommended